toc/working-groups/app-runtime-interfaces.md

```diff
@@ -67,91 +67,217 @@ areas:
   - cloudfoundry/app-autoscaler-env-bbl-state
   - cloudfoundry/app-runtime-interfaces-infrastructure
 
-- name: Buildpacks
+- name: Buildpacks Content
   approvers:
-  - name: Daniel Mikusa
-    github: dmikusa
-  - name: David O'Sullivan
-    github: pivotal-david-osullivan
-  - name: Arjun Sreedharan
-    github: arjun024
-  - name: Brayan Henao
-    github: brayanhenao
   - name: Ryan Moran
     github: ryanmoran
+  - name: Forest Eckhardt
+    github: ForestEckhardt
+  - name: Victoria Campbell
+    github: TisVictress
+  repositories:
+  - cloudfoundry/docs-buildpacks
+  - cloudfoundry/example-sidecar-buildpack
+
+- name: Buildpacks Dependencies
+  approvers:
+  - name: Brayan Henao
+    github: brayanhenao
   - name: Sophie Wigmore
     github: sophiewigmore
+  - name: Rob Dimsdale-Zucker
+    github: robdimsdale
+  repositories:
+  - cloudfoundry/binary-builder
+
+- name: Buildpacks Go
+  approvers:
+  - name: Ryan Moran
+    github: ryanmoran
   - name: Forest Eckhardt
     github: ForestEckhardt
-  - name: Victoria Campbell
-    github: tisvictress
-  - name: Timothy Hitchener
-    github: thitch97
   - name: Rob Dimsdale-Zucker
     github: robdimsdale
-  reviewers:
+  repositories:
+  - cloudfoundry/go-buildpack
+  - cloudfoundry/go-buildpack-release
+
+- name: Buildpacks Java
+  approvers:
+  - name: Daniel Mikusa
+    github: dmikusa
+  - name: David O'Sullivan
+    github: pivotal-david-osullivan
   - name: Anthony Dahanne
```

Comment:
Anthony should remain a reviewer. Any promotion to approver should be in a separate PR.

Reply:
Sorry, that was a mistake due to Anthony being added after this PR was created.